### PR TITLE
Add accessible popovers to vulnerability alerts

### PR DESCRIPTION
**Pull request by drewgillies**
Addresses https://github.com/NuGet/NuGetGallery/issues/8623.

image

### CODE REVIEW

File: src/NuGetGallery/Scripts/gallery/page-display-package.js [hunk 1]
```diff
@@ -170,4 +170,18 @@ $(function () {
...
+    // Add popover to warning icons
+    var popoverFunction = function () {
+        var checkmarkImage = $(this);
+        checkmarkImage.popover({ trigger: 'hover focus' });
+        checkmarkImage.click(function () {
+            checkmarkImage.popover('show');
+            setTimeout(function () {
+                    checkmarkImage.popover('destroy');
+                },
+                1000);
```
Comment (shishirx34) [inline src/NuGetGallery/Scripts/gallery/page-display-package.js:183]: Is 1 second enough for pop over dismissal? Is that a standard timeout value, do we use the same values across our site?

**Review by shishirx34 — APPROVED** [review]

File: src/NuGetGallery/Scripts/gallery/page-manage-packages.js [hunk 2]
```diff
@@ -413,6 +413,19 @@
...
+            var checkmarkImage = $(this);
+            checkmarkImage.popover({ trigger: 'hover focus' });
+            checkmarkImage.click(function () {
+                checkmarkImage.popover('show');
+                setTimeout(function () {
+                        checkmarkImage.popover('destroy');
+                    },
+                    1000);
+            });
+        };
```
Comment (shishirx34) [inline src/NuGetGallery/Scripts/gallery/page-manage-packages.js:427]: Its been a while I've worked on UI changes, but don't we have a common.js or something that is loaded by all the scripts where you could add this function in and share it across different JS scripts? 

see `nuget.copyTextToClipboard` function that we use in similar way.

File: src/NuGetGallery/Scripts/gallery/page-display-package.js (hunk 1 above)
Comment (drewgillies) [reply src/NuGetGallery/Scripts/gallery/page-display-package.js:183]: We do--this is what we're using on the existing popover on reserved checkmarks on the package list.

File: src/NuGetGallery/Scripts/gallery/page-manage-packages.js (hunk 2 above)
Comment (drewgillies) [reply src/NuGetGallery/Scripts/gallery/page-manage-packages.js:427]: I'll check it out--thanks!

File: src/NuGetGallery/Scripts/gallery/page-display-package.js (hunk 1 above)
Comment (drewgillies) [reply src/NuGetGallery/Scripts/gallery/page-display-package.js:183]: (note that the timer starts ticking on mouseleave)

File: src/NuGetGallery/Scripts/gallery/page-manage-packages.js (hunk 2 above)
Comment (drewgillies) [reply src/NuGetGallery/Scripts/gallery/page-manage-packages.js]: Yes, that worked well. I've included this change.

**Pull request merged**